tutorials/math/random_number_generation.rst

```diff
@@ -42,26 +42,56 @@ In global scope, you can find a :ref:`randomize()
 once when your project starts to initialize the random seed.** Calling it
 multiple times is unnecessary and may impact performance negatively.
 
-Putting it in your main scene script's ``_ready()`` method is a good choice::
+Putting it in your main scene script's ``_ready()`` method is a good choice:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _ready():
         randomize()
 
+ .. code-tab:: csharp
+
+    public override void _Ready()
+    {
+        GD.Randomize();
+    }
+
 You can also set a fixed random seed instead using :ref:`seed()
 <class_@GDScript_method_seed>`. Doing so will give you *deterministic* results
-across runs::
+across runs:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _ready():
         seed(12345)
         # To use a string as a seed, you can hash it to a number.
         seed("Hello world".hash())
 
+ .. code-tab:: csharp
+    
+    public override void _Ready()
+    {
+        GD.Seed(12345);
+        GD.Seed((ulong)"Hello world".GetHashCode());
```

Comment:
`"String".Hash()` Doesn't work, but there's a fix for 3.2.4 [here](https://github.com/godotengine/godot/pull/45603/commits/baac70c27efb661e2d3f4127210e7ec991793078#diff-97d1e2000c9580108462e6126ed842c784b02a383bad2bb118e841a5cf061a53), in the mean time, C#'s `"string".GetHashCode()` does the same thing.